packages/x-date-pickers/src/DateCalendar/DateCalendar.tsx

```diff
@@ -88,10 +88,11 @@ type DateCalendarComponent = (<TDate>(
 ) => React.JSX.Element) & { propTypes?: any };
 
 /**
- *
  * Demos:
  *
- * - [Date Picker](https://mui.com/x/react-date-pickers/date-picker/)
+ * - [DatePicker](https://mui.com/x/react-date-pickers/date-picker/)
+ * - [DateCalendar](https://mui.com/x/react-date-pickers/date-calendar/)
+ * - [Validation](https://mui.com/x/react-date-pickers/validation/)
```

Comment:
Could we add `DateCalendar` to the list of `components` on `validation.md` to make this correct?
The page does have a demo with this component. 🤔